### PR TITLE
bugfix: Auto imports in worksheets in Scala 3

### PR DESCRIPTION
**Pull request by jkciesluk**
Previously the import would be inserted above using directives

fixes https://github.com/scalameta/metals/issues/6027

### CODE REVIEW

**Comment by kasiaMarek** [comment]
I think @tgodzik was suggesting to skip any top-level comments, that are not right above `class`/`object`/etc., saying that potentially there can be an license at the top.

File: mtags/src/main/scala/scala/meta/internal/pc/ScriptFirstImportPosition.scala [hunk 1]
```diff
@@ -26,95 +22,120 @@ import scala.meta._
...
-        skipPrefixesOffset(ammHeaders, it, None)
-          .getOrElse(startOffset)
+  def ammoniteScStartOffset(text: String): Option[Int] =
+    scriptStartOffset(text, "/*<start>*/")
 
-      offset + 1
-    }
-  }
+  def scalaCliScStartOffset(text: String): Option[Int] =
+    scriptStartOffset(text, "/*<script>*/")
```
Comment (tgodzik) [inline mtags/src/main/scala/scala/meta/internal/pc/ScriptFirstImportPosition.scala:34]: Aren't both the same now?

**Review by tgodzik — APPROVED** [review]
LGTM

File: mtags/src/main/scala/scala/meta/internal/pc/ScriptFirstImportPosition.scala (hunk 1 above)
Comment (jkciesluk) [reply mtags/src/main/scala/scala/meta/internal/pc/ScriptFirstImportPosition.scala:34]: ScalaCLI uses `<script>` as a marker, where Ammonite uses `<start>`

**Pull request merged**